test/core/integration/fuzz/lockup-base/cancelMultiple.t.sol

```diff
@@ -3,14 +3,19 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { Solarray } from "solarray/src/Solarray.sol";
 
-import { ISablierLockup } from "src/core/interfaces/ISablierLockup.sol";
+import { ISablierLockupBase } from "src/core/interfaces/ISablierLockupBase.sol";
 import { Lockup } from "src/core/types/DataTypes.sol";
 
-import { CancelMultiple_Integration_Shared_Test } from "../../shared/lockup/cancelMultiple.t.sol";
+import { Lockup_Integration_Shared_Test } from "../../shared/lockup/Lockup.t.sol";
+
+abstract contract CancelMultiple_Integration_Fuzz_Test is Lockup_Integration_Shared_Test {
+    Lockup.Model internal lockupModel;
```

Comment:
we can move this in `Integration_Test` so that we can massively dry fy :))

Reply:
Sounds good. But why would it massively dry fy?

Reply:
refer to my PR